tests/Umbraco.Tests.UnitTests/Umbraco.Cms.Api.Management/Services/Signs/IsProtectedSignProviderTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Moq;
+using NUnit.Framework;
+using Umbraco.Cms.Api.Management.Services.Signs;
+using Umbraco.Cms.Api.Management.ViewModels.Tree;
+using Umbraco.Cms.Core.Models.Entities;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Tests.UnitTests.Umbraco.Cms.Api.Management.Services.Signs;
+
+[TestFixture]
+internal class IsProtectedSignProviderTest
```

Comment:
Should pluralize this class name (and the name of the file on disk).  There may be only one test now, but we could add more, and this aligns with other test class names.